style.css

```diff
@@ -1 +1,92 @@
-/*  Add your styling here */
+* {
+    box-sizing: border-box;
+    margin: 0;
+    padding: 0;
+}
+
+body {
+    display: grid;
```

Comment:
Do not change the display of the body. wrap your body content in some root contain er and that will have this style